packages/blocks/src/root-block/widgets/linked-doc/config.ts

```diff
@@ -79,16 +89,20 @@ export const getMenus: (ctx: {
   const filteredDocList = docMetas
     .filter(({ id }) => id !== doc.id)
     .filter(({ title }) => isFuzzyMatch(title, query));
+  const MAX_DOCS = 6;
```

Comment:
I think it would be better to make `maxDisplay` configurable in `LinkedDocOptions`

Reply:
Thanks! Maybe this will be configurable inside block specs in the feature.